src/pages/projects/ProjectsFilterAndSort.tsx

```diff
@@ -94,6 +98,12 @@ export default function ProjectsFilterAndSort({
         <Select.Option value="createdAt">
           <Trans>Date created</Trans>
         </Select.Option>
+        <Select.Option value="currentBalance">
+          <Trans>Current balance</Trans>
+        </Select.Option>
+        <Select.Option value="paymentsCount">
+          <Trans># of payments</Trans>
```

Comment:
Maybe just `Payments` will be okay here

```suggestion
          <Trans># of payments</Trans>
```